ovirt_imageio/_internal/server.py

```diff
@@ -177,3 +184,17 @@ def _drop_privileges(self):
         os.initgroups(self.config.daemon.user_name, gid)
         os.setgid(gid)
         os.setuid(uid)
+
+    def _add_ticket(self, ticket):
+        log.info("Initial ticket: %s", ticket)
+        self.auth.add(self._read_ticket(ticket))
+
+    def _read_ticket(self, ticket):
+        try:
+            with open(ticket, 'r') as f:
+                return json.loads(f.read())
+        except ValueError as e:
+            raise errors.InvalidTicket(
+                f"Ticket is not in a json format: {e}") from e
```

Comment:
We don't know that, maybe:

    f"Cannot parse ticket {ticket}: {e}"

Reply:
I was trying to be consistent with the error given by the ticket Handler: https://github.com/oVirt/ovirt-imageio/blob/master/ovirt_imageio/_internal/handlers/tickets.py#L53